src/Notes.pro

```diff
@@ -41,16 +41,15 @@ HEADERS  += \
 FORMS += $$PWD/mainwindow.ui
 RESOURCES += $$PWD/images.qrc
 
-unix:!macx {
-    isEmpty (PREFIX) {
-        PREFIX = /usr
-    }
-
-    BINDIR = $$PREFIX/bin
-    DATADIR =$$PREFIX/share
-
-    target.path = $$BINDIR
-    INSTALLS += target
+linux:!android {
```

Comment:
Create a symlink from $$Prefix/bin/notes to /usr/bin/notes when PREFIX is not /usr